pkg/lambdaworks/lambdaworks_test.go

```diff
@@ -167,3 +167,27 @@ func TestFeltDiv4Error(t *testing.T) {
 		t.Errorf("TestFeltDiv4Error failed. Expected: %v, Got: %v", expected, result)
 	}
 }
+
+func TestFeltIs0(t *testing.T) {
+	f_zero := lambdaworks.FeltZero()
+
+	expected := true
+
+	result := f_zero.IsZero()
+
+	if result != expected {
+		t.Errorf("TestFeltIs0 failed. Expected: %v, Got %v", expected, result)
+	}
+}
+
+func TestFeltis0Error(t *testing.T) {
+	f_one := lambdaworks.FeltFromUint64(1)
+
+	expected := false
+
+	result := f_one.IsZero()
+
+	if result == expected {
```

Comment:
Test should compare if result != expected to throw an error, otherwise is correct